layouts/partials/head/includes.html

```diff
@@ -1,7 +1,7 @@
 <link rel='icon' href='{{ ( or .Site.Params.assets.favicon "favicon.ico" ) | relURL }}'>
 <link rel='stylesheet' href='{{ print "/assets/css/" .Site.Data.assets.main.css | relURL  }}'>
 
-{{- if .Site.Params.settings.rtl -}}
+{{- if eq .Site.Language.LanguageDirection "rtl" -}}
```

Comment:
Please, don't remove the check for existing option, that can potentially break existing sites using that option.

```suggestion
{{- if or ( .Site.Params.settings.rtl ) ( eq .Site.Language.LanguageDirection "rtl" ) -}}
```

Let's keep it for backward compatibility.